iconsets/system/default/icondef.xml

```diff
@@ -735,4 +740,69 @@
         <object mime='image/png'>mic_rec.png</object>
     </icon>
 
+    <!-- conferenceloggerplugin icon -->
```

Comment:
hm I need to look at this more closely. I believed we had a mechanism to load icons from plugins though iconsets api.
I understand it was the the "*.in" file, but looks like a forgotten artifact.